### PR TITLE
Feat #488 : Benchmarks based on the pure JS Date.now()

### PR DESCRIPTION
**Pull request by sushantdhiman**
Implements the `benchmark` feature using the `getTime` and return the query execution time in `milliseconds`. It doesn't depend upon the `node-microtime`

### CODE REVIEW

File: lib/dialects/mssql/query.js [hunk 1]
```diff
@@ -30,7 +30,14 @@ Query.prototype.run = function(sql, parameters) {
...
   this.sql = sql;
 
-  this.sequelize.log('Executing (' + (this.connection.uuid || 'default') + '): ' + this.sql, this.options);
+  //do we need benchmark for this query execution
+  var benchmark = this.sequelize.options.benchmark || this.options.benchmark;
+
+  if (!benchmark) {
+    this.sequelize.log('Executing (' + (this.connection.uuid || 'default') + '): ' + this.sql, this.options);
+  } else {
+    var queryBegin = (new Date()).getTime();
```
Comment (mickhansen) [inline lib/dialects/mssql/query.js]: `Date.now()` does the same and is a bit shorter.

Comment (sushantdhiman) [inline lib/dialects/mssql/query.js]: Ok

**Comment by sushantdhiman** [comment]
Changed to `Date.now()`

**Comment by sushantdhiman** [comment]
Thanks :)

**Comment by sushantdhiman** [comment]
@mickhansen Can you please add this feature into changelog when releasing next minor / patch version.

**Comment by tornillo** [comment]
Hi, @sushantdhiman!

I propose to pass the bench-time as the second argument to the logger function.
This will allow to do something like this:
```js
function logger (sql, time) {
	if (time => 100/*ms*/) {
		// log to file or do any other action
	}
}
```
This will be especially useful for production environment.
What do you think about this?

**Comment by sushantdhiman** [comment]
Hi @tornillo 

I think thats a good idea. There is already an issue( #2494) related to feature somewhat like this. Current feature has already been merged so I cant change it. But you can send new PR targeting #2494 :)

**Comment by tornillo** [comment]
@sushantdhiman I had not noticed this issue:)
Ok, I have created a new PR https://github.com/sequelize/sequelize/pull/5282 with the changes discussed.

**Pull request merged**